modules/axhal/src/platform/aarch64_common/boot.rs

```diff
@@ -130,6 +118,8 @@ unsafe extern "C" fn _start() -> ! {
         mov     sp, x8
 
         bl      {switch_to_el1}         // switch to EL1
+        adrp    x0, {boot_pt_l0}
+        adrp    x1, {boot_pt_l1}
```

Comment:
Declare `fn init_boot_page_table()` in `boot.rs` and call `crate::platform::mem::init_boot_page_table`, to avoid pass arguments in assembly code.